dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -126,8 +126,7 @@ JavaClass loadJavaClass(String className) throws ClassNotFoundException {
   /** Returns true if {@code className} is available in the system class loader. */
   boolean isSystemClass(String className) {
     try {
-      if (className.startsWith("[")) {
-        // Array class
+      if (isArrayClass(className)) {
```

Comment:
will we catch a reference to an array of a missing class?

Reply:
Yes, but not from array class reference. When there's a reference to array of a class, there should be a reference to the class in the same class file.